pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -280,3 +288,30 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	return config, nil
 }
+
+// ensureDockerAPIVersionForParallelImagePulls returns true if the docker API version
+// is >=1.21, else false.
+func ensureDockerAPIVersionForParallelImagePulls(dh *docker.Helper, version string) bool {
```

Comment:
It seems to me that it would be better to add this kind of function to github.com/fsouza/go-dockerclient or  it would require to create an interface (may be with just Version function as you said) with docker client like what kube has with docker manager.